compiler/il/OMRBlock.cpp

```diff
@@ -1545,9 +1546,9 @@ OMR::Block::setInstructionBoundaries(uint32_t startPC, uint32_t endPC)
    }
 
 void
-OMR::Block::addExceptionRangeForSnippet(uint32_t startPC, uint32_t endPC)
+OMR::Block::addExceptionRangeForSnippet(OMR::Compilation *comp, uint32_t startPC, uint32_t endPC)
```

Comment:
If this is only using the compilation for access to trMemory, it should probably just take one of those directly instead.